### ISSUE TITLE
Instabug crashes on startup

### ISSUE THREAD

**Issue by GlisboaDev** [Instabug/Instabug-React-Native#343]
We're receiving random crashes from Instabug initialization on our production app. They're random crashes which I don't have the steps to reproduce, but attached the logs we have bellow.

### Steps to Reproduce the Problem
Happens randomly, no specific steps

### Expected Behavior
Not crash the app

### Actual Behavior
Crashes the app on Instabug initialization

### Instabug integration code
```
 const setup = () => {
    Instabug.startWithToken("",
      [Instabug.invocationEvent.shake]
    );
    Instabug.setWelcomeMessageMode(Instabug.welcomeMessageMode.disabled);
    Instabug.setCrashReportingEnabled(false);
    NetworkLogger.setEnabled(false);
    BugReporting.setPromptOptionsEnabled(false, true, true);
    BugReporting.onInvokeHandler(() => {
      logInfo();
    });

    Instabug.setEnabledAttachmentTypes(true, false, false, false);
  };
```

### SDK Version
`8.4.4`

### React Native, iOS and Android Versions
`"react-native": "0.58.4"`

### Device Model
iPad Mini OS 9.3.5
iPad 2 OS 9.3.5

### Logs

```
RCTFatalException: Exception '-[NSRunLoop performBlock:]: unrecognized selector sent to instance 0x1553e6b0' was thrown while invoking startWithToken on target Instabug with params ( [KEY XXXXXXX], ( 1 ) ) callstack: ( 0 CoreFoundation 0x24ddb933 <redacted> + 150 1 libobjc.A.dylib 0x24576e17 objc_exception_throw + 38 2 CoreFoundation 0x24de12b5 <redacted> + 0 3 CoreFoundation 0x24ddeee1 <redacted> + 700 4 CoreFoundation 0x24d0a238 _CF_forwarding_prep_0 + 24 5 123Loadboard 0x002aa4a3 _ZN8facebook5react11JSIExecutor21defaultTimeoutInvokerERKNSt3__18functionIFvvEEENS3_IFNS2_12basic_stringIcNS2_11char_traitsIcEENS2_9allocatorIcEEEEvEEE + 802938 6 CoreFoundation 0x24de04c4 <redacted> + 68 7 CoreFoundation 0x24d0687d <redacted> + 292 8 CoreFoundation 0x24d0a2f7 <redacted> + 50 9 123Loadboard 0x0024a913 _ZN8facebook5react11JSIExecutor21defaultTimeoutInvokerERKNSt3__18functionIFvvEEENS3_IFNS2_12basic_stringIcNS2_11char_traitsIcEENS2_9allocatorIcEEEEvEEE + 410858 10 123Loadboard 0x00250ce5 _ZN8facebook5react11JSIExecutor21defaultTimeoutInvokerERKNSt3__18functionIFvvEEENS3_IFNS2_12basic_stringIcNS2_11char_traitsIcEENS2_9allocatorIcEEEEvEEE + 436412 11 123Loadboard 0x00250967 _ZN8facebook5react11JSIExecutor21defaultTimeoutInvokerERKNSt3__18functionIFvvEEENS3_IFNS2_12basic_stringIcNS2_11char_traitsIcEENS2_9allocatorIcEEEEvEEE + 435518 12 libdispatch.dylib 0x24949823 <redacted> + 10 13 libdispatch.dylib 0x2494980f <redacted> + 22 14 libdispatch.dylib 0x24957ba9 <redacted> + 1524 15 CoreFoundation 0x24d9db6d <redacted> + 8 16 CoreFoundation 0x24d9c067 <redacted> + 1574 17 CoreFoundation 0x24ceb229 CFRunLoopRunSpecific + 520 18 CoreFoundation 0x24ceb015 CFRunLoopRunInMode + 108 19 Foundation 0x25534045 <redacted> + 268 20 Foundation 0x255511ab <redacted> + 86 21 123Loadboard 0x002dad67 _ZN8facebook5react11JSIExecutor21defaultTimeoutInvokerERKNSt3__18functionIFvvEEENS3_IFNS2_12basic_stringIcNS2_11char_traitsIcEENS2_9allocatorIcEEEEvEEE + 1001790 22 123Loadboard 0x00021337 123Loadboard + 45879 23 UIKit 0x293cb785 <redacted> + 392 24 UIKit 0x295f32d1 <redacted> + 3080 25 UIKit 0x295f7285 <redacted> + 1588 26 UIKit 0x2960b83d <redacted> + 36 27 UIKit 0x295f47b3 <redacted> + 134 28 FrontBoardServices 0x2647ec07 <redacted> + 18 29 FrontBoardServices 0x2647eab9 <redacted> + 224 30 FrontBoardServices 0x2647edb9 <redacted> + 48 31 CoreFoundation 0x24d9ddff <redacted> + 14 32 CoreFoundation 0x24d9d9ed <redacted> + 452 33 CoreFoundation 0x24d9bd5b <redacted> + 794 34 CoreFoundation 0x24ceb229 CFRunLoopRunSpecific + 520 35 CoreFoundation 0x24ceb015 CFRunLoopRunInMode + 108 36 UIKit 0x293c4c3d <redacted> + 524 37 UIKit 0x293bf189 UIApplicationMain + 144 38 123Loadboard 0x00022865 123Loadboard + 51301 39 libdyld.dylib 0x24993873 <redacted> + 2 )RCTAssert.m:145
```

**Comment by HeshamMegid** [comment]
@GlisboaDev thanks for the report. We're looking into it.

**Comment by GlisboaDev** [comment]
@HeshamMegid Any updates on it? This is happening in a huge frequency (282 this weekend.)

**Comment by GlisboaDev** [comment]
Not sure if it's exactly the same problem, but just noticed this when debugging the app locally:
```
=================================================================
Main Thread Checker: UI API called on a background thread: -[UIApplication applicationState]
PID: 57415, TID: 14589839, Thread name: com.apple.CoreMotion.MotionThread, Queue name: com.apple.root.default-qos.overcommit, QoS: 0
Backtrace:
4   libobjc.A.dylib                     0x00000001af7e7534 <redacted> + 56
5   CoreMotion                          0x00000001b6070040 CoreMotion + 307264
6   CoreMotion                          0x00000001b6070574 CoreMotion + 308596
7   CoreMotion                          0x00000001b6070484 CoreMotion + 308356
8   CoreMotion                          0x00000001b60a1c64 CoreMotion + 511076
9   CoreMotion                          0x00000001b60a1cc4 CoreMotion + 511172
10  CoreFoundation                      0x00000001b0580484 <redacted> + 28
11  CoreFoundation                      0x00000001b057fd68 <redacted> + 276
12  CoreFoundation                      0x00000001b057ab28 <redacted> + 1016
13  CoreFoundation                      0x00000001b057a418 CFRunLoopRunSpecific + 452
14  CoreFoundation                      0x00000001b057b17c CFRunLoopRun + 84
15  CoreMotion                          0x00000001b60a15fc CoreMotion + 509436
16  libsystem_pthread.dylib             0x00000001b01f4974 <redacted> + 132
17  libsystem_pthread.dylib             0x00000001b01f48d0 _pthread_start + 52
18  libsystem_pthread.dylib             0x00000001b01fcddc thread_start + 4
2019-08-05 11:49:44.452125-0400 123Loadboard[57415:14589839] [reports] Main Thread Checker: UI API called on a background thread: -[UIApplication applicationState]
PID: 57415, TID: 14589839, Thread name: com.apple.CoreMotion.MotionThread, Queue name: com.apple.root.default-qos.overcommit, QoS: 0
Backtrace:
4   libobjc.A.dylib                     0x00000001af7e7534 <redacted> + 56
5   CoreMotion                          0x00000001b6070040 CoreMotion + 307264
6   CoreMotion                          0x00000001b6070574 CoreMotion + 308596
7   CoreMotion                          0x00000001b6070484 CoreMotion + 308356
8   CoreMotion                          0x00000001b60a1c64 CoreMotion + 511076
9   CoreMotion                          0x00000001b60a1cc4 CoreMotion + 511172
10  CoreFoundation                      0x00000001b0580484 <redacted> + 28
11  CoreFoundation                      0x00000001b057fd68 <redacted> + 276
12  CoreFoundation                      0x00000001b057ab28 <redacted> + 1016
13  CoreFoundation                      0x00000001b057a418 CFRunLoopRunSpecific + 452
14  CoreFoundation                      0x00000001b057b17c CFRunLoopRun + 84
15  CoreMotion                          0x00000001b60a15fc CoreMotion + 509436
16  libsystem_pthread.dylib             0x00000001b01f4974 <redacted> + 132
17  libsystem_pthread.dylib             0x00000001b01f48d0 _pthread_start + 52
18  libsystem_pthread.dylib             0x00000001b01fcddc thread_start + 4
WARN : BSG_KSCrashSentry.c (109): BSG_KSCrashType
```
And I always get this stack exactly on the line:
`[Instabug startWithToken:@"" invocationEvents:IBGInvocationEventShake];`

**Comment by HeshamMegid** [comment]
@GlisboaDev the stack trace you shared is coming from the Core Motion framework. It's a bug in the framework that Apple has acknowledged, and it's safe to ignore. More details [here](https://github.com/Instabug/Instabug-iOS/issues/298#issuecomment-485663403).

We're still working on a fix for the other issue. We're sorry it's taking that long.

**Comment by HeshamMegid** [comment]
@GlisboaDev just letting you know that we fixed this in a1c845ba4c5f71e9ff14d11d8e1bde34e28cf9e0. This fix will be included in the version we are releasing later today.

**Comment by HeshamMegid** [comment]
We just released v8.5.3 that includes this fix.
